src/main/java/core/basesyntax/services/ParseService.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.services;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class ParseService {
+    public Date stringToDate(String inputDate) {
+        try {
+            return new SimpleDateFormat("dd.MM.yyyy").parse(inputDate);
+        } catch (ParseException e) {
+            throw new RuntimeException("Can not parse string to java.util.date", e);
+        }
+    }
+}
```

Comment:
you can use this logic directly, just define this field  `DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");`

Reply:
ok